etc/ci/check_dynamic_symbols.py

```diff
@@ -0,0 +1,26 @@
+#/usr/bin/env python
+
+import sys, re, subprocess
+from sets import Set
+
+symbol_regex = re.compile("D  \*UND\*\t(.*) (.*)$")
+allowed_symbols = Set(['unshare', 'malloc_usable_size'])
+actual_symbols = Set()
+
+objdump_output = subprocess.check_output([
+        'arm-linux-androideabi-objdump',
+        '-T',
+        'target/arm-linux-androideabi/debug/libservo.so']
+    ).split('\n')
+
+for line in objdump_output:
+    m = symbol_regex.search(line)
+    if m != None:
+        actual_symbols.add(m.group(2))
+
+difference = allowed_symbols.difference(actual_symbols)
```

Comment:
I believe we need to do `actual_symbols.difference(allowed_symbols)`, which can written with syntax sugar as `actual_symbols - allowed_symbols`. This will take the actual symbols and remove the symbols we know are OK, leaving the symbols that are not OK.